components/breadcrumb-item.js

```diff
@@ -0,0 +1,30 @@
+const template = `
+<li class="c-breadcrumb__item" :class="classes">
+    <a v-if="!current" :href="href">
+      <slot></slot>
+    </a> 
+    <slot v-else></slot>
+</li>
+`;
+
+const BreadcrumbItem = {
+  template,
+  props: {
+    current: {
+      type: Boolean,
+      default: false
+    },
+    href: {
+      type: String,
+      default: '#'
```

Comment:
Please make this `javascript:void(0);` instead

Reply:
Ok I see this makes sense